grails-app/services/org/broadinstitute/orsp/StorageProviderService.groovy

```diff
@@ -441,10 +441,9 @@ class StorageProviderService implements Status {
      * @return A GoogleCredential from json secrets.
      */
     private GoogleCredential getCredential() {
+
         if (!credential) {
-            File configFile = GrailsResourceUtils.getFile(
-                    GrailsResourceUtils.toURI(this.getClass().getResource(storageConfiguration.config)),
-                    "GCS Config")
+            File configFile = new File(storageConfiguration.config);
```

Comment:
Question: Is this necessary for deployment? If not, revert and file a separate PR for the simplification.

Reply:
I think it is, to get configuration for storage.